st/clitests/jclient_spec.py

```diff
@@ -756,7 +756,20 @@ def get_response_elements(response):
 
     JClientTest('Jclient can delete bucket').delete_bucket("seagatebucket").execute_test().command_is_successful()
 
+# ************ TEST: Multiple INIT-MPU ***********
```

Comment:
TEST: Multipart initiate of same object twice

Reply:
done